config/routes.rb

```diff
@@ -2,6 +2,7 @@
   root to: redirect('users')
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create]
+    resources :comments, only: [:new, :create]
```

Comment:
- [ ] Kindly add the routes, controller, and view for the likes so that a user would be able to like a post according to the requirements 👍 
![Annotation 2022-06-18 214728](https://user-images.githubusercontent.com/85113567/174456648-7e468da0-0c6a-4fde-8bc1-0abbf24ebfc6.jpg)

Reply:
Thanks, already done!

Reply:
- [ ] As the previous reviewer requested, you should allow users add likes to posts, currently your app does not count likes on a post as expected, the number of likes should be counted on a particular post e.g likes on Facebook or Instagram posts  you see numbers of likes on a particular post

Reply:
Thanks I thought I just had to add likes to the first post